VTCareService/src/main/java/edu/vt/cs/vtcare/vtcareservice/dao/LoginDao.java

```diff
@@ -0,0 +1,38 @@
+package edu.vt.cs.vtcare.vtcareservice.dao;
+import edu.vt.cs.vtcare.vtcareservice.db.VTCareJDBC;
+
+import java.sql.*;
+
+public class LoginDao {
+    private final Connection connection;
+
+    private static final String LOGIN =
```

Comment:
Rename the variable appropriately to reflect that it is storing an SQL statement. like LOGIN_SQL

Reply:
Its completely understandable that the query is for login, and not something else. Adding redundancy in the names does not add to the readability.

Reply:
I guess you are confusing one concept with other. If we go on to categorize such things as redundancy half of the source code in the world's software would cease to exist. These things are devised upon much research and experience of the software engineers around the world. There's nothing debatable about it.